trustchain_design.md

```diff
@@ -29,11 +29,11 @@
 
 When a user intents to share a resource with another user or group of users, *Tanker Core* encrypts the *resource* with the symmetric [Resource Encryption Key] it generates; this [Resource Encryption Key] is then encrypted asymmetrically for the listed recipients, using their public keys. The challenge in this scheme is being sure that the [Resource Encryption Key] is shared with the right *user*, without forcing *users* to physically exchange their public keys.
 
-The basic role of a *Trustchain* is to provide public key distribution between *users* without being able to tamper with anything or access any private data.
+The primary role of the *Trustchain* server is to provide public key distribution between *users* without being able to tamper with anything or access any private data. This guaranty is provided by the *Trustchain* itself.
 
 ## Users and devices
 
-Devices are the basic entities we considered when designing the *Trustchain*. A *device* is an entity owned by a *user*. It stores a [Device Signature Key Pair] and a [Device Encryption Key Pair]. In practice, *devices* can be *users*' web browsers, mobile phones, desktop applications, etc.
+Devices are the simplest entities we considered when designing the *Trustchain*. A *device* is an entity owned by a *user*. It stores a [Device Signature Key Pair] and a [Device Encryption Key Pair]. In practice, *devices* can be *users*' web browsers, mobile phones, desktop applications, etc.
```

Comment:
it's not really simple, a key publish is simple. basic was more appropriate here imho

Reply:
first works too. I don't like `basic`, it's not the right adjectives in this context.